e2e-tests/tests/config-router/06-restart-mysql.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      kubectl -n "${NAMESPACE}" rollout restart deployment config-router-router
```

Comment:
what's the purpose of this step?

Reply:
After creating CM for the config, we need to restart router pods so the config is applied.